lib/services/broker/wsBrokerClient.js

```diff
@@ -20,13 +21,19 @@ function WSBrokerClient (brokerType, options, pluginsManager, notifyOnListen) {
     socket: null,
     connected: null,
     state: 'disconnected',
-    retryInterval: options.retryInterval
+    retryInterval: options.retryInterval || 1000
   };
   this.server = {
     host: options.host,
     port: options.port
   };
+  
   this.handlers = {};
+  this.onConnectHandlers = [];
```

Comment:
sure, [here you have](https://github.com/kuzzleio/kuzzle-plugin-cluster/blob/master/lib/cluster/slaveNode.js#L55).